modules/distr-tracing-tempo-config-default.adoc

```diff
@@ -4,147 +4,58 @@
 
 :_mod-docs-content-type: REFERENCE
 [id="distr-tracing-tempo-config-default_{context}"]
-= Default configuration options
+= Introduction to TempoStack configuration parameters
 
 The `TempoStack` custom resource (CR) defines the architecture and settings to be used when creating the {TempoShortName} resources. You can modify these parameters to customize your {TempoShortName} implementation to your business needs.
```

Comment:
```suggestion
The `TempoStack` custom resource (CR) defines the architecture and settings that you can use when creating the {TempoShortName} resources. You can modify these parameters to customize your {TempoShortName} implementation to your business needs.
```

I realize these are out of scope for this PR, but some suggestions for the future:
* "to be used" -> "that you can use" to make this active
* I would only use the {TempoShortName} once if at all here - IMHO, it is confusing to read the long name with (Tempo). I think it would be better to just use this once and then use "Tempo" for the rest of the doc maybe, but you might have someone ask customers their feelings about it 🙂

Reply:
Thank you, Ashleigh, here's how I changed it:

1. `to be used when` > `for` :slightly_smiling_face: 
2. I dropped the second `{TempoShortName}` mention.

I'm glad you had a look at it.